vite.config.js

```diff
@@ -3,6 +3,6 @@ import react from '@vitejs/plugin-react';
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  base: '/2023-02-Orleans-Spe-P3/', // TODO: use the right repo name
+  base: '/2023-02-Orleans-Spe-P3-B/', // TODO: use the right repo name
```

Comment:
```suggestion
  base: '/2023-02-Orleans-Spe-P3-B/',
```
Le commentaire peut être supprimé du coup